index.js

```diff
@@ -1,28 +1,103 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+const {
+    getAllRecipes,
+    findUserByEmail,
+    addNewUser
+} = require('./database');
```

Comment:
This is another option for importing, but functionally the same. In my opinion, it loses a bit of the 'readability' aspect of where these functions come from versus saving a variable for the object.